qt/scientific_interfaces/ISISReflectometry/GUI/Instrument/QtInstrumentView.cpp

```diff
@@ -93,6 +95,15 @@ void QtInstrumentView::disconnectSettingsChange(QCheckBox &edit) {
 
 void QtInstrumentView::onSettingsChanged() { m_notifyee->notifySettingsChanged(); }
 
+void QtInstrumentView::browseToCalibrationFile() {
+  auto calibrationFilePath = QFileDialog::getOpenFileName(this, QString(), QString(), tr("Data Files (*.dat)"));
```

Comment:
As mentioned, I think we can use an interface to get the file path rather than direct use of QFileDialog, so here I suggest we just notify the presenter and let it do all the work, and then just call back to update the view with the result.